.github/workflows/winget-submission.yml

```diff
@@ -1,38 +1,15 @@
-name: Submit MSEdgeRedirect to winget
+name: Publish to WinGet
 
 on:
   release:
     types: [published]
   workflow_dispatch:
```

Comment:
```suggestion
on:
  release:
    types: [released]
```

Reply:
Thanks @sitiom 👍

Reply:
I suggest reverting the change and letting @rcmaehl decide for himself.